### PR TITLE
change MacOS version, default shell, vars file

### PR DESCRIPTION
**Pull request by odebretseni**

### CODE REVIEW

File: tasks/install-additional-software.yml [hunk 1]
```diff
@@ -1,4 +1,12 @@
 ---
+- name: Install Applications from Homebrew
+  homebrew:
+    name: '{{ item }}'
+    options: force
+  loop:
+    - python@3
+  become: false
```
Comment (BrunIF) [inline tasks/install-additional-software.yml]: Needles. All roles by default running from user.

File: tasks/install-oh-my-zsh.yml [hunk 2]
```diff
@@ -11,7 +11,12 @@
...
-- name: Setting default shell
+- name: Change oh-my-zsh standart theme
+  lineinfile:
+    path: /Users/{{ work_user }}/.zshrc
+    regexp: '^ZSH_THEME='
+    line: 'ZSH_THEME="kafeitu"'
+
+- name: Set zsh as default shell
+  user: name={{ work_user }} shell=/bin/zsh
   become: true
```
Comment (BrunIF) [inline tasks/install-oh-my-zsh.yml]: needles for user

File: meta/main.yml [hunk 3]
```diff
@@ -11,8 +11,8 @@ galaxy_info:
   platforms:
     - name: MacOS
```
Comment (BrunIF) [inline meta/main.yml]: OS name should be `MacOSX`

File: meta/main.yml [hunk 4]
```diff
@@ -11,8 +11,8 @@ galaxy_info:
...
       versions:
-        - "Catalina"
-        - "Mojave"
+        - "10.14"
```
Comment (BrunIF) [inline meta/main.yml]: Try without quotes

**Review by BrunIF — CHANGES_REQUESTED** [review]

File: tasks/install-additional-software.yml [hunk 5]
```diff
@@ -1,4 +1,11 @@
 ---
+- name: Install Applications from Homebrew
+  homebrew:
+    name: '{{ item }}'
+    options: force
+  loop:
```
Comment (BrunIF) [inline tasks/install-additional-software.yml:6]: Haven't sense. Because you install one app.

**Review by BrunIF — CHANGES_REQUESTED** [review]

File: tasks/install-additional-software.yml (hunk 5 above)
Comment (odebretseni) [reply tasks/install-additional-software.yml:6]: but there is a ready cycle in case something else appears

**Review by BrunIF — APPROVED** [review]

**Pull request merged**